### ISSUE TITLE
How to use it with with-info addon?

### ISSUE THREAD

**Issue by danielo515** [rbardini/storybook-addon-playroom#11]
Hello, and thanks for this awesome addon, it really allows for fast prototyping.
I think you are aware of the with-info addon. It automatically generates some documentation for each story that contains the story source and a table of properties.
If I add your plugin, then the sources changed drastically, I guess is related to what your addon injects.
This is the expected appearance:
![image](https://user-images.githubusercontent.com/2270425/80263609-6e789e80-8691-11ea-91cb-6b4a56894d76.png)

This is the actual output:

![image](https://user-images.githubusercontent.com/2270425/80263537-425d1d80-8691-11ea-9872-05fa9166fb42.png)

I have your addon the first on on the list of addons/decorators:

```javascript
import { addDecorator } from '@storybook/react';
import { withConsole } from '@storybook/addon-console';
import { withPlayroom } from 'storybook-addon-playroom';
import { withInfo } from '@storybook/addon-info';

addDecorator(withPlayroom); // before any other decorators
addDecorator(withInfo); 
addDecorator((storyFn, context) => withConsole()(storyFn)(context));
```

It will be awesome to be able to use both plugins.

**Comment by rbardini** [comment]
Hi @danielo515!

It may be incompatible with `addon-info`, since both addons rely on React element serialization. Have you tried adding the decorator after `withInfo`?

Alternatively, I've tested it with `addon-docs` and it worked as expected:

![addon-docs](https://user-images.githubusercontent.com/874370/80640880-56739700-8a64-11ea-8409-01a61c7f91ba.png)

I highly recommend using `addon-docs` instead—`addon-info` has been [deprecated in its favor](https://github.com/storybookjs/deprecated-addons/tree/master/addons/info). It uses a [Webpack loader](https://github.com/storybookjs/storybook/tree/master/lib/source-loader) to extract the story source (which gives more reliable results), is well-maintained and looks much better 🙂

**Comment by danielo515** [comment]
Thanks for your answer @rbardini .
Yes, I tried adding it later and I get very similar results.

I already tried addon-docs, and it looks very nice, but they have a problem with detecting prop-types where addon-info does it just fine, so I had to come back to addon-info. I will create some more components ( I was just using the example ones) and try addon-docs again, and if they render my actual components nicely, then I will use it instead.

Regards
